### ISSUE TITLE
Extra :// on Latex math equations.

### ISSUE THREAD

**Issue by ringosham** [noties/Markwon#136]
* **Markwon version**: 3.0.1
* **Markwon Latex extension version**: 3.0.1

1. Please specify expected/actual behavior
The equations should render properly instead of this.
![Screenshot_20190618-221703](https://user-images.githubusercontent.com/7325740/59720471-d9a64880-9216-11e9-808b-569374e991a0.jpg)
2. Please specify conditions/steps to reproduce (layout, code, markdown used, etc)
```
TextView text = findViewById(R.id.test);
Markwon markdown = Markwon.builder(this).usePlugin(ImagesPlugin.create(this)).usePlugin(JLatexMathPlugin.create(50)).build();
markdown.setMarkdown(text, "$$a+b=c$$");
```
Simply use this code in an activity would achieve this effect.
App's minimum SDK version: 21
Compiled SDK version: 28
Android version: Pie (9)

**Comment by noties** [comment]
Hello @ringosham !

Can confirm, will release a new version with this bug fixed tomorrow

**Comment by ringosham** [comment]
Cool. Thanks for the fast response.

**Comment by noties** [comment]
@ringosham I have published `3.0.2` version with this issue fixed
